application/views/admin/labels/exportmulti_view.php

```diff
@@ -33,3 +33,22 @@
     </div>
 </div>
 
+<script type='text/javascript'>
+    /** If one or more elements are selected, then the export button will be enabled.
+     *  otherwise its disabled by default. */
+    let isSelected    = false;
+    let exportButton  = document.getElementById('exportButton');
+
+    $('select').change(function() {
+        $('#labelsets option:selected').each(function() {
```

Comment:
`isSelected = $('#labelsets option:selected').length` > 0 ? think it's more quick :)